code/src/lib/Buttons.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import styled from "styled-components/macro";
+
+export const Button = ({title, type, disabled, className, onClick, key, background}) => {
+
+  return (
+    <Btn 
+      type={type} 
+      disabled={disabled} 
+      className={className}
+      onClick={onClick}
+      key={key}
+      background={background}
+    >
+      {title}
+    </Btn>
+  )
+}
+
+const Btn = styled.button`
```

Comment:
We can see that you modify the button-styling in different places. Consider making different buttons for different purposes to make it easier to change styling by going to one place :)